web/impact/impact/tests/test_api_routes.py

```diff
@@ -111,6 +111,41 @@ def test_api_object_get(self):
             response_dict = json.loads(response.content)
             self.assertIn("is_visible", response_dict.keys())
 
+    def test_api_related_object_list_get(self):
```

Comment:
Not a blocker, but this name is not informative. I don't know what this test is actually intended to test. 
Mentioning it because we do a lot of this and it'd be nice to get out of this habit.

Reply:
So I thought this was actually quite informative.  It gets the list of related objects as opposed to the tests above it which are just for lists of objects...